### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by PaulWessel**
**Description of proposed changes**

Updated ChangeLog for GMT 6.1.  Please, hoping @GenericMappingTools/core and @GenericMappingTools/gmt-contributors can have a look in case I have left anything important out.  @seisman should probably expand on the *seis* upgrades. The 6.1 train is header for a release cliff this weekend, so pronto please.

### CODE REVIEW

File: doc/rst/source/changes.rst [hunk 1]
```diff
@@ -4,6 +4,83 @@
...
 
+New Features in GMT 6.1
+=======================
+
+GMT 6.1 may be a minor revision to 6.0 but packs quite a punch. For general
+changes, we mention
+
+ #. Updated remote global data sets: Earth relief, land/ocean masks, and day/night imagery. The larger
+    grid files (5x5 arc minutes and smaller resolutions) are now tiled and faster to download.
+ #. Add MVT (Mapbox Vector Tile) as another GDAL URL dataset.
```
Comment (joa-quim) [inline doc/rst/source/changes.rst:15]: Better remove this one. We do nothing with the vector tiles yet. It was just to anticipate future use.

File: doc/rst/source/changes.rst [hunk 2]
```diff
@@ -4,6 +4,83 @@
...
+#. :doc:`grdinfo`: Now **-C** also appends registration as last output column (0 = gridline, 1 = pixel registration).
+#. :doc:`grdmath`: New operators **DAYNIGHT** (for day/night terminator), **BLEND** (blend two grids using the weights
+   from a third), **DOT** (dot product), and **RGB2HSV**, and **HSV2RGB** for color manipulations.
+#. :doc:`grdtrack`: Determine central peak in all crossections with **-F** (requires **-C**); let **-E+c** continue
+   a track if next line is a direct continuation of previous line.
+#. :doc:`pscontour`: Better handling of contour file that can now have unique angles and pens per contour.
+#. :doc:`movie`: Add **-E** for an optional title sequence (with or without fading in/out), **-K** for fade in and
+   fade out for main animation sequence, **-Sb** and **-Sf** can now take a PostScript layer instead of a script,
+   and **-P** for adding one of six progress indicators.
+#. :doc:`sample1d`: Adds a smoothing cubic spline via **-Fs**\ *p* (for a fit parameter *p*), with optional weights (**-W**).
```
Comment (joa-quim) [inline doc/rst/source/changes.rst:76]: Add entry for nearneighbor -Nn saying that it calls the GDALʻs nearest neighbor algorithm

File: doc/rst/source/changes.rst [hunk 3]
```diff
@@ -4,6 +4,83 @@
...
+ #. Add modifier **+v** for a *vertical* oblique Equator in -JO [horizontal].
+ #. New **-B** modifier **+i** for placing internal frame annotations
+ #. New **-B** modifier **+f** to turn on fancy geographic annotations.
+ #. New polar projection (**-JP**) modifiers  (**+f**\|\ **r**\|\ **t**\|\ **z**) adds new 
+    capabilities for annotating azimuths, depths or radii.
+ #. Revise verbosity default levels and their names and abbreviations.
+ #. Add Web-Mercator as new sphere that can be selected.
+ #. Explore adding long-format GMT options (e.g., **--region**\ =\ *w/e/s/n*).
+ #. Allow both **-i** and **-o** to specify an open-ended list of columns to end of record.
+ #. API improvements to support the GMT/MEX, pyGMT, and GMT.jl environments.
```
Comment (seisman) [inline doc/rst/source/changes.rst:33]:
```suggestion
 #. API improvements to support the GMT/MEX, PyGMT, and GMT.jl environments.
```

File: doc/rst/source/changes.rst [hunk 4]
```diff
@@ -4,6 +4,83 @@
...
+#. :doc:`grd2kml`: New option **-W** for adding contour overlays. Also rebuilt for global grids as well as
+   to write PNG or JPG directly (depending on transparency) without going via *PostScript* conversion (only
+   required if **-W** is used).
+#. :doc:`grdcontour`: Better handling of contour file that can now have unique angles and pens per contour.
+#. :doc:`grdconvert`: Enable scaling/translation services on output with **-Z**.
+#. :doc:`grdfill`: Implement minimum-curvature spline infill with **-As**.
+#. :doc:`grdfilter`: Let filter width optionally be a grid with variable widths.
+#. :doc:`grdgradient`: Add support for ambient light in **-N**, as in **-E**, and therefore via **-I**
+   in :doc:`grdimage` and :doc:`grdview`.
+#. :doc:`grdinfo`: Now **-C** also appends registration as last output column (0 = gridline, 1 = pixel registration).
```
Comment (seisman) [inline doc/rst/source/changes.rst:67]:
```suggestion
#. :doc:`grdinfo`: Now **-C** also appends registration and gtype as last two output columns (0 = gridline, 1 = pixel registration; 0=Cartesian, 1=geographic).
```

File: doc/rst/source/changes.rst (hunk 1 above)
Comment (PaulWessel) [reply doc/rst/source/changes.rst:15]: done

File: doc/rst/source/changes.rst (hunk 2 above)
Comment (PaulWessel) [reply doc/rst/source/changes.rst:76]: done

File: doc/rst/source/changes.rst (hunk 3 above)
Comment (PaulWessel) [reply doc/rst/source/changes.rst:33]: done

File: doc/rst/source/changes.rst (hunk 4 above)
Comment (PaulWessel) [reply doc/rst/source/changes.rst:67]: done

**Pull request merged**